nextcloudappstore/core/api/v1/release/info.xsd

```diff
@@ -32,9 +32,6 @@
                             maxOccurs="1"/>
                 <xs:element name="repository" type="repository" minOccurs="0"
                             maxOccurs="1"/>
-                <xs:element name="discussion" type="discussion-url"
```

Comment:
If you validate it using the exact schema, it will. As for the app store: we've got some xslt in place that simply strips out unknown tags before validation.